mods/sp/rules/infantry.yaml

```diff
@@ -611,9 +617,16 @@ MARAUDER:
 	DamageMultiplier@MarauderBuff:
 		Modifier: 75
 		RequiresCondition: deployed
+	AIDeployHelper:
+		DeployChance: 100
+		DeployTrigger: Attack
+		DeployTicks : 0
 	RejectsOrders@deployment:
 		Reject: AttackMove, AssaultMove
-		RequiresCondition: deployed
+		RequiresCondition: !AImicroManage && deployed
+	RejectsOrders@deploymentAI:
```

Comment:
This should be marked asa  to-do, because this is a nasty hack and only required due to lack of updates in ABrandau/OpenRA.

Reply:
Ok buddy. Is AI deploy helper or elsewhere get upgrade for this kind of unit situation?

Reply:
https://github.com/AttacqueSuperior/Engine/commit/411343cfbe2f79bd91e2630343915b9c271b925b

Reply:
Finally a conditional one.

Reply:
However, I need AI to move the "Deploy as fixed turret" type units at any situation. Conditional AIDHelper can help, but everything here.